templates/rally-beyond-the-paywall/measurements/advertisement.1.schema.json

```
@@ -0,0 +1,89 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$comment": "Please do not edit this schema by hand. Generate it instead by running npm run build:schema.",
+  "mozPipelineMetadata": {
+    "expiration_policy": {
+      "delete_after_days": 558
+    }
+  },
+  "type": "object",
+  "properties": {
+    "type": {
+      "type": "string",
+      "description": "the type of event recorded. Either WebScience.advertisements, WebScience.articleContents, WebScience.pageNav, or WebScience.pageNavSensitive",
+      "enum": [
+        "WebScience.advertisements",
+        "WebScience.articleContents",
+        "WebScience.pageNav",
+        "WebScience.pageNavSensitive",
+        "WebScience.totalTiming"
+      ]
+    },
+    "userId": {
+      "type": "string",
+      "description": "a unique ID associated with a participant."
+    },
+    "visitId": {
+      "type": "string",
+      "description": "a unique ID associated with a page visit."
+    },
+    "body": {
+      "type": "object",
+      "description": "This object contains two properties listed below about the <body> tag of the page.  This is needed to normalize the size of advertisments against full page size.",
+      "properties": {
+        "body.clientHeight": {
+          "type": "integer",
+          "description": "The calculated height in pixels of the <body> tag"
+        },
+        "body.clientWidth": {
+          "type": "integer",
+          "description": "The calculated width in pixels of the <body> tag"
+        }
+      }
+    },
+    "ads": {
+      "type": "array",
+      "description": "This array contains an object per advertisement found",
+      "items": {
+        "ads.id": {
```

Each of these properties includes an `ads.`-prefix that should be removed.

```suggestion
        "id": {
```